<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="uiActionOpen"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="uiActionZoom_In"/>
    <addaction name="uiActionZoom_Out"/>
    <addaction name="uiActionNormal_Size"/>
    <addaction name="uiActionFit_to_Window"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="uiActionAbout"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>View Mode</string>
    </property>
    <addaction name="uiActionLeft"/>
    <addaction name="uiActionRight"/>
    <addaction name="uiActionRed_Green"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuMode"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="uiActionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="uiActionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="uiActionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="uiActionNormal_Size">
   <property name="text">
    <string>Normal Size</string>
   </property>
  </action>
  <action name="uiActionFit_to_Window">
   <property name="text">
    <string>Fit to Window</string>
   </property>
  </action>
  <action name="uiActionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="uiActionLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="uiActionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="uiActionRed_Green">
   <property name="text">
    <string>Red-Green</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
